src/main/java/org/fungover/mmotodo/team/TeamService.java

```diff
@@ -0,0 +1,128 @@
+package org.fungover.mmotodo.team;
+
+import jakarta.persistence.EntityNotFoundException;
+import jakarta.transaction.Transactional;
+import jakarta.validation.Valid;
+import org.fungover.mmotodo.exception.TeamNotFoundException;
+import org.fungover.mmotodo.task.Task;
+import org.fungover.mmotodo.task.TaskRepository;
+import org.fungover.mmotodo.user.User;
+import org.fungover.mmotodo.user.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.*;
+
+
+@Service
+public class TeamService {
+    private final TeamRepository teamRepository;
+
+    private final UserRepository userRepository;
+
+    private final TaskRepository taskRepository;
+
+    @Autowired
+    public TeamService(TeamRepository teamRepository, UserRepository userRepository, TaskRepository taskRepository) {
+
+        this.teamRepository = teamRepository;
+        this.userRepository = userRepository;
+        this.taskRepository = taskRepository;
+    }
```

Comment:
Good use of injection, however again `@Autowired` is not needed. Here is also an additional empty line 27 that can be removed.